experiment/simulation/simulation.html

```diff
@@ -0,0 +1,319 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+	<meta charset="UTF-8">
+	<meta http-equiv="X-UA-Compatible" content="IE=edge">
+	<meta name="viewport" content="width=device-width, initial-scale=1.0">
+
+	<title>Document</title>
+	<link rel="stylesheet" href="https://cdn.jsdelivr.net/gh/virtual-labs/virtual-style@0.0.7-b/css/style.min.css">
+	<link rel="stylesheet" href="css/main.css">
+	<!-- <script src="https://cdn.jsdelivr.net/npm/interactjs/dist/interact.min.js"></script> -->
+</head>
+
+<body>
+	<div class="container" style="padding-top:3%">
+		<div class="columns is-centered">
+			<div class="column is-one-quarter v-instruction-title">
+				Tasks
+			</div>
+			<div class="column">
+				<div class="v-tabs">
+					<ul>
+						<li class="is-active" id="Task1" onclick="changeTabs(event)">
+							<a>
+								Task 1
+							</a>
+						</li>
+					</ul>
+				</div>
+			</div>
+		</div>
+
+		<div class="columns is-centered">
+			<div class="column">
+				<div class="v-instruction-container">
+					<div class="v-instruction-title" id="TaskTitle">Half Adder</div>
+					<div class="v-instruction-text" id="TaskDescription">
+						Implement a 1-bit half adder
+					</div>
+				</div>
+			</div>
+		</div>
+	</div>
+	<div class="columns is-centered">
+		<div class="column is-one-quarter" style="margin:auto;text-align:center">
+		</div>
+		<div class="column"></div>
+		<div class="column is-one-third">
+			<a href="simulation.html"><button class="v-button">Restart simulation</button></a>
+		</div>
+	</div>
+	<div class="columns is-centered">
+		<div class="column is-2-desktop is-4-tablet is-12-mobile">
+			<div class="v-datalist-container components-list">
+				<div class="v-datalist-title">Components</div>
+				<div class="v-datalist-row">
+					<div class="columns is-centered" id="toolbar">
+						<div class="column is-one-half">
+							<div class="component-button AND" onclick="Add(event)">AND</div>
+						</div>
+						<div class="column is-one-half">
+							<div class="component-button XOR" onclick="Add(event)">XOR</div>
+						</div>
+					</div>
+				</div>
+
+			</div>
+		</div>
+		<div class="column is-7-desktop is-12-tablet is-12-mobile">
+			<div id="circuit-board">
+				<div id="working-area">
+					<div id="zero-zero">
+						<div id="path1">
```

Comment:
Can we use more meaningful names than path1 - 4?

Reply:
The simulation was rewritten in a different library (GSAP) so this whole file has changed, I have integrated that with other files too now.